.circleci/config.yml

```diff
@@ -284,6 +284,8 @@ commands:
             cp *_version versions || true
       - store_artifacts:
           path: /tmp/logs
+      - store_test_results:
+          path: /tmp/test-results
```

Comment:
does this short-circuit properly if the directory DNE? since this step will also run for lte integ tests

Reply:
Yeah the step will not fail even if there is no file in the directory